### ISSUE TITLE
Sync Gateway Accelerator is reporting itself as Sync Gateway

### ISSUE THREAD

**Issue by sethrosetter** [couchbase/sync_gateway#1462]
when launching sg_accel, and pinging :4984, the response is:

```
{
"couchdb": "Welcome",
"vendor": {
"name": "Couchbase Sync Gateway",
"version": 1.2
},
"version": "Couchbase Sync Gateway/master(8d14249)"
}
```

This could cause some confusion in user experience.

**Comment by sethrosetter** [comment]
Still not fixed in 1.4-40

`GET :4985` on sg_accel node

```
{
"ADMIN": true,
"couchdb": "Welcome",
"vendor": {
"name": "Couchbase Sync Gateway",
"version": 1.4
},
"version": "Couchbase Sync Gateway/1.4(40;35563f3)"
}
```

**Comment by ajres** [comment]
It looks like the latest changes to `couchbase\build` project have not been propagated onto the Jenkins server.

Looking at: `http://mobile.jenkins.couchbase.com/view/Sync_Gateway/job/sgw-unix-build/configure`

I can see that `${TARGET_DIR}/cbbuild/scripts/jenkins/mobile/sgw_unix_build.sh`

Is not the latest version from the `couchbase\build` repo

The Jenkins job does not appear to pull the build scripts from a git repo. It looks like there is another process to upgrade the build.

@sethrosetter @tleyden Do you know if I need to ssh into the Jenkins box and upgrade manually or if there is another process to follow?

**Comment by ceejatec** [comment]
The build repo is part of the manifest, and so it is updated as part of unpacking the source tarball. The problem is that the build repo is locked to a specific SHA in the sync_gateway manifest.

I can't currently think of a reason that this would be a good thing, so I'll propose a change to unlock cbbuild.

**Comment by ajres** [comment]
Ah, I didn't know it was in the manifest.

I'm wondering if it was pinned to allow us to re-build old releases, which may need to use older versions of the build scripts?

**Comment by sethrosetter** [comment]
@ajres Please give me heads up when this is about to go in so I can brace for any runtime validation I may have to change

**Comment by tleyden** [comment]
I think after @ceejatec's change, this should be fixed.  Going to proactively close -- @sethrosetter can you re-open if you still see the issue?

**Comment by ajres** [comment]
The commit hash referenced in the manifest was on the original branch and master.

I have created a new branch from the commit hash and just copied over the changes for this ticket.

I have pushed to gerrit for review, but I'm not sure what the branch will merge to @tleyden you are assigned for review can we touch base before merge the change.

**Comment by tleyden** [comment]
@ceejatec this is blocked on https://github.com/couchbase/sync_gateway/pull/2276#issuecomment-277136727 -- that's why this got assigned to you

**Comment by tleyden** [comment]
> I have pushed to gerrit for review, but I'm not sure what the branch will merge to @tleyden you are assigned for review can we touch base before merge the change.

No longer relevant.  This was to isolate this change from couchbase/build master branch, but we decided as a team to get things working against couchbase/build master (the issue described in https://github.com/couchbase/sync_gateway/pull/2276#issuecomment-277136727)

**Comment by adamcfraser** [comment]
Depends on https://github.com/couchbase/sync_gateway/issues/2324

**Comment by adamcfraser** [comment]
@sethrosetter This should be ready to verify.

**Comment by sethrosetter** [comment]
@adamcfraser What build is is this change a part of?

**Comment by adamcfraser** [comment]
1.4-268 should include this change.  It's got Ceej's commit to unlock the build in the SG manifest, and looking at the manifest used for the build, it's using the latest from the build repo.

http://latestbuilds.hq.couchbase.com/couchbase-sync-gateway/1.4/1.4-268/sync_gateway-1.4-268-manifest.xml

**Comment by sethrosetter** [comment]
This is fixed
